app/views/admin/editions/_locale_fields.html.erb

```diff
@@ -2,15 +2,17 @@
   <fieldset class="foreign-language">
     <div class="checkbox">
       <%= label_tag :create_foreign_language_only, nil, class: [:checkbox, "js-toggle-scheduled-publication-date-picker"] do %>
-        <%= check_box_tag :create_foreign_language_only, "1", form.object.primary_locale != "en" %> Create a foreign language only news article
+        <%= check_box_tag :create_foreign_language_only, "1", form.object.primary_locale != "en" %> <%="Create a foreign language only #{edition.model_name.human.downcase}"%>
       <% end %>
     </div>
     <div class="form-group foreign-language-select js-hidden">
       <%= form.label :primary_locale, 'Document language' %>
       <div class="form-inline add-label-margin">
         <%= form.select :primary_locale, options_for_foreign_language_locale(edition), {}, {class: 'form-control input-md-6'} %>
       </div>
-      <p class="warning">Warning: News stories without an English version cannot have other translations.</p>
+      <% if edition.is_a?(NewsArticle) %>
+        <p class="warning">Warning: News stories without an English version cannot have other translations.</p>
+      <% end %>
```

Comment:
Should this be the case for document collections too?

Reply:
I don't believe it is possible for DocumentCollections to have a translation at all, so I don't think we need to warn.